From 44a5f1ecca308f67d3ffea61ae8426415812789d Mon Sep 17 00:00:00 2001
From: Marcelo Leitner <mleitner@redhat.com>
Date: Tue, 21 Oct 2014 16:21:27 -0400
Subject: [redhat] configs: config adjusts for nftables/netfilter

Message-id: <20141021162127.22888.29810.stgit@localhost.localdomain>
Patchwork-id: 98020
O-Subject: [RHEL7 net PATCH 123/124] redhat: config adjusts for nftables/netfilter
Bugzilla: 1148041 1155088
RH-Acked-by: Florian Westphal <fwestpha@redhat.com>

RHEL specific

Enables nftables new bits.

Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/CONFIG_IP6_NF_NAT b/redhat/configs/generic/CONFIG_IP6_NF_NAT
new file mode 100644
index 0000000..9796b0b
--- /dev/null
+++ b/redhat/configs/generic/CONFIG_IP6_NF_NAT
@@ -0,0 +1 @@
+CONFIG_IP6_NF_NAT=m
diff --git a/redhat/configs/generic/CONFIG_IP_NF_NAT b/redhat/configs/generic/CONFIG_IP_NF_NAT
new file mode 100644
index 0000000..d2aa272
--- /dev/null
+++ b/redhat/configs/generic/CONFIG_IP_NF_NAT
@@ -0,0 +1 @@
+CONFIG_IP_NF_NAT=m
diff --git a/redhat/configs/generic/CONFIG_NFT_BRIDGE_META b/redhat/configs/generic/CONFIG_NFT_BRIDGE_META
new file mode 100644
index 0000000..93dbf6f
--- /dev/null
+++ b/redhat/configs/generic/CONFIG_NFT_BRIDGE_META
@@ -0,0 +1 @@
+CONFIG_NFT_BRIDGE_META=m
diff --git a/redhat/configs/generic/CONFIG_NFT_BRIDGE_REJECT b/redhat/configs/generic/CONFIG_NFT_BRIDGE_REJECT
new file mode 100644
index 0000000..1ec341f
--- /dev/null
+++ b/redhat/configs/generic/CONFIG_NFT_BRIDGE_REJECT
@@ -0,0 +1 @@
+CONFIG_NFT_BRIDGE_REJECT=m
diff --git a/redhat/configs/generic/CONFIG_NFT_MASQ_IPV4 b/redhat/configs/generic/CONFIG_NFT_MASQ_IPV4
new file mode 100644
index 0000000..5a5ec1c
--- /dev/null
+++ b/redhat/configs/generic/CONFIG_NFT_MASQ_IPV4
@@ -0,0 +1 @@
+CONFIG_NFT_MASQ_IPV4=m
diff --git a/redhat/configs/generic/CONFIG_NFT_MASQ_IPV6 b/redhat/configs/generic/CONFIG_NFT_MASQ_IPV6
new file mode 100644
index 0000000..d1e703f
--- /dev/null
+++ b/redhat/configs/generic/CONFIG_NFT_MASQ_IPV6
@@ -0,0 +1 @@
+CONFIG_NFT_MASQ_IPV6=m
-- 
1.7.1